W3ChampionsStatisticService/Matches/MatchesController.cs

```diff
@@ -26,8 +26,13 @@ public MatchesController(IMatchRepository matchRepository, MatchQueryHandler mat
         int offset = 0,
         int pageSize = 100,
         GameMode gameMode = GameMode.Undefined,
-        int season = 18)
+        int season = 0)
```

Comment:
There is a season 0. Either set default null or perhaps -1.
```suggestion
        int? season = null)
```

Reply:
fixed